Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/FloatSliderSetting.kt

```diff
@@ -56,9 +56,6 @@ open class FloatSliderSetting : SliderSetting {
         get() = floatSetting.float
 
     open fun setSelectedValue(settings: Settings, selection: Float) {
-        floatSetting.setFloat(
-            settings,
-            BigDecimal((selection).toDouble()).round(MathContext(3)).toFloat()
-        )
+        floatSetting.setFloat(settings, selection)
```

Comment:
I'm not sure why this rounding was being done. @t895, any comment?

Reply:
I think this was an incorrect assumption about how precise our floating point settings were. This was from when I was trying to get float sliders to not break from precision issues for the color correction settings but when I try to use any float slider, things work fine in this build. Seems fine for me.